webapp/src/Controller/API/BalloonController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+namespace App\Controller\API;
+
+
+use App\Entity\Contest;
+use App\Service\BalloonService;
+use FOS\RestBundle\Controller\Annotations as Rest;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Swagger\Annotations as SWG;
+use Doctrine\ORM\QueryBuilder;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Intl\Exception\NotImplementedException;
+
+/**
+ * @Rest\Route("/api/v4/contests/{cid}/balloons", defaults={ "_format" = "json" })
+ * @Rest\Prefix("/api/contests/{cid}/balloons")
+ * @Rest\NamePrefix("balloons_")
+ * @SWG\Tag(name="Balloons")
+ * @SWG\Parameter(ref="#/parameters/cid")
+ * @SWG\Response(response="404", ref="#/definitions/NotFound")
+ * @SWG\Response(response="401", ref="#/definitions/Unauthorized")
+ * @Security("is_granted('ROLE_JURY') or is_granted('ROLE_API_READER') or is_granted('ROLE_BALLOON')")
```

Comment:
I think at some point we should remove jury here and in other places, since jury had api reader automatically

Reply:
It helps to be explicit if you'd ever want to change the definition of roles.

Reply:
In that case we should add it to some endpoints that don’t have it. Also sometimes in code (“show more if we have role x”). I’ll check the code.